Lib/__future__.py

```diff
@@ -96,7 +96,7 @@ def getMandatoryRelease(self):
         """Return release in which this feature will become mandatory.
 
         This is a 5-tuple, of the same form as sys.version_info, or, if
-        the feature was dropped, is None.
+        the feature was dropped, or it's release date is undetermined, is None.
```

Comment:
```suggestion
        the feature was dropped, or the release date is undetermined, is None.
```

Otherwise the phrase sounds like "it is release date is undetermined"

Reply:
Sorry English is my second language (my first is python 🐍)